lib/docs/token-return-types.md

```diff
@@ -0,0 +1,71 @@
+# Token Return Types
+
+When acquiring tokens, MSAL JS v1 considers three different Token Request types:
+
+| Token Type | Definition |
+| ------- | -------- |
+| **token** | Indicates a request will be made for an Access Token |
+| **id_token** | Indicates a request will be made for an ID Token |
+| **id_token token** | Indicates that both an ID Token and Access Token will be requested |
+
+The following table describes how MSAL JS v1 determines which token request type is set and therefore what kind of token is returned by the API in question. 
+
+| Scenarios | ClientId as Only Scope | No Scopes | Access Token Scopes with No ClientId | Access Token Scopes with ClientId
+| ------- | ------- | -------- | --------- | -------- |
+| **loginRedirect/Popup** | id_token | id_token |id_token token | id_token token
+| **acquireTokenInteractive** | id_token | Error | token | id_token token 
+| **acquireTokenSilent** | id_token | Error | token | id_token token
```

Comment:
cc @hamiltonha Once `error` doc is ready, please link to the exact error thrown here. Can you still update if the errors are distinct? If not leave it be until @hamiltonha's doc is ready.